bolt-delegations-cli/src/config.rs

```diff
@@ -0,0 +1,62 @@
+use clap::{Parser, Subcommand, ValueEnum};
+
+/// A CLI tool to generate signed delegation messages for BLS keys.
+#[derive(Parser, Debug, Clone)]
+pub struct Opts {
+    #[clap(subcommand)]
+    pub command: Commands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum Commands {
+    /// Generate delegation messages.
+    Generate {
+        /// The source of the validator key (local or keystore).
+        #[clap(long, env = "SOURCE")]
+        source: SourceType,
+
+        /// Path to the keystore file or private key, depending on the source.
+        #[clap(long, env = "KEY_PATH")]
+        key_path: String,
```

Comment:
As suggested by @mempirate also, usually private keys are not stored in plain text. You can take inspiration from the `SigningOpts` in the sidecar and make the user choose between a path to the keystore, along with its password, or a private key passed directly.